### DISCUSSION TITLE
某些配置参数是否考虑不重启热更新

### DISCUSSION THREAD

**Discussion by viccom** [EasyTier/EasyTier#295] [Ideas]
对某些参数是否可以设计为不重启程序在线生效，如：
  -n, --proxy-networks <PROXY_NETWORKS>
          export local networks to other peers in the vpn
      --exit-nodes [<EXIT_NODES>...]
          exit nodes to forward all traffic to, a virtual ipv4 address, priority is determined by the order of the
          list
      --enable-exit-node
          allow this node to be an exit node, default is false
      --manual-routes [<MANUAL_ROUTES>...]
          assign routes cidr manually, will disable subnet proxy and
          wireguard routes propogated from peers. e.g.: 192.168.0.0/16
以上仅仅是举例，对于必须重启程序才能生效的参数，提示必须重启程序。此功能是否有意义？是否方便实现？负责人来定夺。

**Comment by KKRainbow** [comment]
重启一下影响非常小，简单又有效。
做热更新又复杂又容易出错
